builtins/davix/CMakeLists.txt

```diff
@@ -51,7 +52,8 @@ ExternalProject_Add(DAVIX
     -DCMAKE_OSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}
     -DLIB_SUFFIX=
   LOG_BUILD 1 LOG_CONFIGURE 1 LOG_DOWNLOAD 1 LOG_INSTALL 1
-  BUILD_BYPRODUCTS ${DAVIX_LIBRARIES})
+  BUILD_BYPRODUCTS ${DAVIX_LIBRARIES}
+  TIMEOUT ${timeoutProject})
```

Comment:
Why add a new variable, why not simply `TIMEOUT 600`? 600 sounds good btw, that's 10 minutes for download operations, good enough.